docs/en_US/sqls/streams.md

```diff
@@ -37,6 +37,8 @@ CREATE STREAM
 | StrictValidation     | true | To control validation behavior of message field against stream schema. See [Strict Validation](#Strict Validation) for more info. |
 | CONF_KEY | true | If additional configuration items are requied to be configured, then specify the config key here. See [MQTT stream](../rules/sources/mqtt.md) for more info. |
 | SHARED | true | Whether the source instance will be shared across all rules using this stream |
+| TIMESTAMP | true | The field to represent the event's timestamp. If specified, the rule will run with event time. Otherwise, it will run with processing time. Please refer to [timestamp management](./windows.md#timestamp-management) for details. |
```

Comment:
Change `TIMESTAMP` to `TIMESTAMP_FIELD` to make it more specific?

Reply:
I suggest no because it will cause breaking API changes